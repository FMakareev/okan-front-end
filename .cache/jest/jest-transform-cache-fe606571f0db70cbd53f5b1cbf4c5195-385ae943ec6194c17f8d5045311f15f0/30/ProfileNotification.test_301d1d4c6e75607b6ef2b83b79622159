1815ca2eb4723516e6b2d3a641155146
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _enzyme = require("enzyme");

var _StyledThemeProvider = require("../../../../styles/StyledThemeProvider");

var _ProfileNotification = require("./ProfileNotification/");

describe('News container initial', function () {
  var props = {
    data: {
      message: '1'
    }
  };
  var newsContainer = (0, _enzyme.shallow)(_react["default"].createElement(_StyledThemeProvider.StyledThemeProvider, null, _react["default"].createElement(_ProfileNotification.ProfileNotification, props))); // передали props

  it('renders properly', function () {
    expect(newsContainer).toMatchSnapshot();
  });
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlByb2ZpbGVOb3RpZmljYXRpb24udGVzdC5qcyJdLCJuYW1lcyI6WyJkZXNjcmliZSIsInByb3BzIiwiZGF0YSIsIm1lc3NhZ2UiLCJuZXdzQ29udGFpbmVyIiwiaXQiLCJleHBlY3QiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7Ozs7QUFBQTs7QUFDQTs7QUFDQTs7QUFFQTs7QUFDQTs7QUFFQUEsUUFBUSxDQUFDLHdCQUFELEVBQTJCLFlBQU07QUFDdkMsTUFBTUMsS0FBSyxHQUFHO0FBQ1pDLElBQUFBLElBQUksRUFBRTtBQUNKQyxNQUFBQSxPQUFPLEVBQUU7QUFETDtBQURNLEdBQWQ7QUFNQSxNQUFNQyxhQUFhLEdBQUcscUJBQ3BCLGdDQUFDLHdDQUFELFFBQ0UsZ0NBQUMsd0NBQUQsRUFBeUJILEtBQXpCLENBREYsQ0FEb0IsQ0FBdEIsQ0FQdUMsQ0FXcEM7O0FBRUhJLEVBQUFBLEVBQUUsQ0FBQyxrQkFBRCxFQUFxQixZQUFNO0FBQzNCQyxJQUFBQSxNQUFNLENBQUNGLGFBQUQsQ0FBTixDQUFzQkcsZUFBdEI7QUFDRCxHQUZDLENBQUY7QUFHRCxDQWhCTyxDQUFSIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IFJlYWN0IGZyb20gJ3JlYWN0JztcclxuaW1wb3J0IHJlbmRlcmVyIGZyb20gJ3JlYWN0LXRlc3QtcmVuZGVyZXInO1xyXG5pbXBvcnQgeyBzaGFsbG93IH0gZnJvbSAnZW56eW1lJztcclxuXHJcbmltcG9ydCB7IFN0eWxlZFRoZW1lUHJvdmlkZXIgfSBmcm9tICcuLi8uLi8uLi8uLi9zdHlsZXMvU3R5bGVkVGhlbWVQcm92aWRlcic7XHJcbmltcG9ydCB7IFByb2ZpbGVOb3RpZmljYXRpb24gfSBmcm9tICcuL1Byb2ZpbGVOb3RpZmljYXRpb24vJztcclxuXHJcbmRlc2NyaWJlKCdOZXdzIGNvbnRhaW5lciBpbml0aWFsJywgKCkgPT4ge1xyXG4gIGNvbnN0IHByb3BzID0ge1xyXG4gICAgZGF0YToge1xyXG4gICAgICBtZXNzYWdlOiAnMScsXHJcbiAgICB9LFxyXG4gIH07XHJcblxyXG4gIGNvbnN0IG5ld3NDb250YWluZXIgPSBzaGFsbG93KFxyXG4gICAgPFN0eWxlZFRoZW1lUHJvdmlkZXI+XHJcbiAgICAgIDxQcm9maWxlTm90aWZpY2F0aW9uIHsuLi5wcm9wc30gLz5cclxuICAgIDwvU3R5bGVkVGhlbWVQcm92aWRlcj4sXHJcbiAgKTsgLy8g0L/QtdGA0LXQtNCw0LvQuCBwcm9wc1xyXG5cclxuICBpdCgncmVuZGVycyBwcm9wZXJseScsICgpID0+IHtcclxuICAgIGV4cGVjdChuZXdzQ29udGFpbmVyKS50b01hdGNoU25hcHNob3QoKTtcclxuICB9KTtcclxufSk7XHJcbiJdfQ==