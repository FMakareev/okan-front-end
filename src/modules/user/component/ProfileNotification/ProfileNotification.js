import React, { useState } from 'react';
import PropTypes from 'prop-types';
import isEmpty from 'lodash/isEmpty';

/**View */
import Text from '@lib/ui/Text/Text';
import Flex from '@lib/ui/Flex/Flex';
import Box from '@lib/ui/Box/Box';
import PaginationPage from '@lib/ui/PaginationPage/PaginationPage';

/** Components */
import ProfileNotificationItem from '../ProfileNotificationItem/ProfileNotificationItem';

export const ProfileNotification = ({ data, pageNumber, pageSize }, props) => {
  console.log(props);
  return (
    <>
      <Box mb={[8]}>
        <Text
          fontSize={6}
          lineHeight={8}
          color={'color7'}
          textAlign={'center'}
          mb={[4]}
          fontFamily={'primary500'}>
          Оповещения
        </Text>
      </Box>
      {data && data.map(item => <ProfileNotificationItem {...item} />)}
    </>
  );
};

ProfileNotification.propTypes = {
  /** message user*/
  message: PropTypes.string,
};

export default ProfileNotification;

// <PaginationPage pageNumber={pageNumber} pageSize={pageSize} data={data} Component={ProfileNotificationItem} />;
