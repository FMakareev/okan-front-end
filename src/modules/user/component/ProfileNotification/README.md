админ:
- без аргмуентов выдает все записи
- с userid выдает записи конкретного пользователя
- с userid и messageread выдает записи конкретного 
пользователя в разных состояниях, если id соответствует админу то не прочитаные меняются на прочитаные

Пользователь:

- без аргмуентов не выдает
- с userid соответствующим авторизованному юзеру выдает все его уведомления,  
прочитаные меняются на прочитаные
- с userid и messageread соответствующим авторизованному юзеру выдает все его уведомления,
  прочитаные меняются на прочитаные
