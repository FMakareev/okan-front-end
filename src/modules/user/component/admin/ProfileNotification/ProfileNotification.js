import React, { Component, Fragment } from 'react';
import styled from 'styled-components';

import Text from '../../../../../components/Text/Text';

const TextStyled = styled(Text)`
  background-color: #f2f2f2;
  border: 1px solid #848484;
  border-radius: 5px;
  padding: 5px;
`;

export class ProfileNotification extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <Fragment>
        <Text fz={6} lh={7} color={'color7'} textAlign={'center'} mb={4}>
          Оповещения
        </Text>

        <TextStyled fz={5} lh={7} color={'color10'} mb={4}>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
          ut labore et dolore magna aliqua. Ut enim ad minim
        </TextStyled>

        <TextStyled fz={5} lh={7} color={'color10'} mb={4}>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
          ut labore et dolore magna aliqua. Ut enim ad minim
        </TextStyled>
      </Fragment>
    );
  }
}

export default ProfileNotification;
