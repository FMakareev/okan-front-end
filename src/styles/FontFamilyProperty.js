/**
 * @params {string} fontFamily - string name of the font name property in the theme
 * @params {object} theme
 * */
export const FontFamilyProperty = ({ fontFamily, theme }) => {
  try {
    if (fontFamily && theme.fontFamily) {
      return `font-family: '${theme.fontFamily[fontFamily]}';`;
    }
  } catch (e) {
    console.error(e);
  }
  return undefined;
};
export default FontFamilyProperty;
