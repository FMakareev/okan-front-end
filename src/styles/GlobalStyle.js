import { createGlobalStyle } from 'styled-components'


export const GlobalStyle = createGlobalStyle`
  *{
    box-sizing: border-box;
  }
  body, html, #app {
    padding: 0;
    margin: 0;
    width: 100%;
    min-height: 100vh;
    background-color: #fff;
  }
`;
