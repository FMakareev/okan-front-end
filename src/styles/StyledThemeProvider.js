import React from 'react';
import PropTypes from 'prop-types';
import { Provider } from 'rebass';

/** Variants */
import { ButtonVariant } from './variants/buttonVariant';
import { ButtonSize } from './variants/buttonSize';
import { ButtonComment } from './variants/buttonComment';
import { TextVariant } from './variants/textVariant';
import { InputSize } from '@lib/styles/variants/InputSize';
import { InputVariant } from '@lib/styles/variants/InputVariant';

export const ColorPallet = {
  color0: '#FFFFFF',
  color1: '#E5E5E5',
  color2: '#DEDEDE',
  color3: '#4F4F4F',
  color4: '#848484',
  color5: '#007FAF',
  color6: '#006699',
  color7: '#00649C',
  color8: '#007CC4',
  color9: '#F97E63',
  color10: '#AEAEAE',
  color11: '#333333',
  color12: '#DF4624',
  color13: '#f2f2f2',
  color14: '#fff3c1',
};

const Space = [
  0,
  2,
  4,
  8,
  12,
  16,
  18,
  20,
  24,
  28,
  32,
  36,
  40,
  44,
  48,
  52,
  56,
  60,
  64,
  68,
  72,
  76,
  80,
];

const Weight = [300, 500, 700];

const Breakpoints = ['1024px', '1200px', '1440px'];

const boxShadow = [
  '2px 4px 8px 0px rgba(138,138,138,0.5)',
  '2px 4px 8px 0px rgba(138,138,138,0.5)',
  '2px 2px 4px 0px rgba(0,127,175,1)',
];

const textAlign = ['left', 'center', 'right'];

export const StyledThemeProvider = ({ children }) => (
  <Provider
    theme={{
      space: Space,
      fontSizes: Space,
      fontWeight: Weight,
      lineHeight: Space,
      boxShadow,
      borderRadius: Space,
      borderColor: ColorPallet,
      breakpoints: Breakpoints,
      colors: ColorPallet,
      variant: {
        buttons: ButtonVariant,
        buttonSize: ButtonSize,
        buttonComment: ButtonComment,
        text: TextVariant,
        inputSize: InputSize,
        inputVariant: InputVariant,
      },
      fontFamily: {
        primary300: 'Museo Sans 300',
        primary500: 'Museo Sans 500',
        primary700: 'Museo Sans 700',
        secondary: 'Circe Regular',
        secondaryBold: 'Circe Bold',
      },
      textAlign: textAlign,
    }}>
    {children}
  </Provider>
);

StyledThemeProvider.propTypes = {
  children: PropTypes.element.isRequired,
};

StyledThemeProvider.defaultProps = {
  children: 'children is empty',
};

export default StyledThemeProvider;
