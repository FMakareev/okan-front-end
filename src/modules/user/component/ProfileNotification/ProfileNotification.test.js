import React from 'react';
import renderer from 'react-test-renderer';
import { StyledThemeProvider } from '../../../../styles/StyledThemeProvider';
import { ProfileNotification } from './ProfileNotification/ProfileNotification';

it('ProfileNotification: Рендерится без ошибок', () => {
  renderer.create(
    <StyledThemeProvider>
      <ProfileNotification />
    </StyledThemeProvider>,
  );
});
