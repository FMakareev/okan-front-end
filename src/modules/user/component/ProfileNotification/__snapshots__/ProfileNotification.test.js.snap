// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`News container initial renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <StyledThemeProvider>
    <ProfileNotification
      data={
        Object {
          "message": "1",
        }
      }
    />
  </StyledThemeProvider>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <ProfileNotification
        data={
          Object {
            "message": "1",
          }
        }
      />,
      "theme": Object {
        "borderColor": Object {
          "color0": "#FFFFFF",
          "color1": "#E5E5E5",
          "color10": "#AEAEAE",
          "color11": "#333333",
          "color12": "#DF4624",
          "color13": "#f2f2f2",
          "color14": "#fff3c1",
          "color2": "#DEDEDE",
          "color3": "#4F4F4F",
          "color4": "#848484",
          "color5": "#007FAF",
          "color6": "#006699",
          "color7": "#00649C",
          "color8": "#007CC4",
          "color9": "#F97E63",
        },
        "borderRadius": Array [
          0,
          2,
          4,
          8,
          12,
          16,
          18,
          20,
          24,
          28,
          32,
          36,
          40,
          44,
          48,
          52,
          56,
          60,
          64,
          68,
          72,
          76,
          80,
        ],
        "boxShadow": Array [
          "2px 4px 8px 0px rgba(138,138,138,0.5)",
          "2px 4px 8px 0px rgba(138,138,138,0.5)",
          "2px 2px 4px 0px rgba(0,127,175,1)",
        ],
        "breakpoints": Array [
          "1024px",
          "1200px",
          "1440px",
        ],
        "colors": Object {
          "color0": "#FFFFFF",
          "color1": "#E5E5E5",
          "color10": "#AEAEAE",
          "color11": "#333333",
          "color12": "#DF4624",
          "color13": "#f2f2f2",
          "color14": "#fff3c1",
          "color2": "#DEDEDE",
          "color3": "#4F4F4F",
          "color4": "#848484",
          "color5": "#007FAF",
          "color6": "#006699",
          "color7": "#00649C",
          "color8": "#007CC4",
          "color9": "#F97E63",
        },
        "fontFamily": Object {
          "primary300": "Museo Sans 300",
          "primary500": "Museo Sans 500",
          "primary700": "Museo Sans 700",
          "secondary": "Circe Regular",
          "secondaryBold": "Circe Bold",
        },
        "fontSizes": Array [
          0,
          2,
          4,
          8,
          12,
          16,
          18,
          20,
          24,
          28,
          32,
          36,
          40,
          44,
          48,
          52,
          56,
          60,
          64,
          68,
          72,
          76,
          80,
        ],
        "fontWeight": Array [
          300,
          500,
          700,
        ],
        "lineHeight": Array [
          0,
          2,
          4,
          8,
          12,
          16,
          18,
          20,
          24,
          28,
          32,
          36,
          40,
          44,
          48,
          52,
          56,
          60,
          64,
          68,
          72,
          76,
          80,
        ],
        "space": Array [
          0,
          2,
          4,
          8,
          12,
          16,
          18,
          20,
          24,
          28,
          32,
          36,
          40,
          44,
          48,
          52,
          56,
          60,
          64,
          68,
          72,
          76,
          80,
        ],
        "textAlign": Array [
          "left",
          "center",
          "right",
        ],
        "variant": Object {
          "buttonComment": Object {
            "comment": Object {
              "border": 0,
              "fill": "#006699",
              "minWidth": "20px",
            },
            "emptyComment": Object {
              "border": 0,
              "fill": "#848484",
              "minWidth": "20px",
            },
            "newComment": Object {
              "border": 0,
              "fill": "#df4624",
              "minWidth": "20px",
            },
          },
          "buttonSize": Object {
            "large": Object {
              "fontFamily": "Circe Bold",
              "fontSize": " 32px",
              "lineHeight": "40px",
              "minWidth": "500px",
              "padding": "8px 60px",
            },
            "medium": Object {
              "fontFamily": "Circe Bold",
              "fontSize": "18px",
              "lineHeight": "24px",
              "minWidth": "250px",
              "padding": "8px 0",
            },
            "small": Object {
              "fontFamily": "Circe Regular",
              "fontSize": " 16px",
              "lineHeight": "16px",
              "padding": "2px 0",
            },
            "toApproval": Object {
              "fontSize": "15px",
              "padding": "2px",
            },
            "xsmall": Object {
              "-moz-padding": "0",
              "fontSize": " 14px",
              "fontWeight": 700,
              "lineHeight": "16px",
              "minWidth": "70px",
              "padding": "2px 0",
            },
          },
          "buttons": Object {
            "disabled": Object {
              "&:hover": Object {
                "backgroundColor": "#00649c57",
                "color": "#fff",
                "fill": "#fff",
              },
              "backgroundColor": "#fff",
              "border": "1px solid #00649c57",
              "color": "#00649c57",
              "fill": "#00649c57",
            },
            "empty": Object {
              "&:hover": Object {
                "color": "#00649C",
                "fill": "#00649C",
                "stroke": "#00649C",
              },
              ":disabled": Object {
                "backgroundColor": "#e5e5e5",
                "fill": "#e5e5e5",
              },
              "backgroundColor": "#ffffff",
              "border": 0,
              "fill": "#848484",
              "stroke": "#848484",
            },
            "large": Object {
              "&:hover": Object {
                "backgroundColor": "#00649C",
                "color": "#ffffff",
                "fill": "#ffffff",
              },
              "backgroundColor": "transparent",
              "border": "2px solid",
              "color": "#00649C",
              "fill": "#00649C",
            },
            "outlineGray": Object {
              "&:hover": Object {
                "borderColor": "#00649C",
                "color": "#00649C",
                "fill": "#00649C",
                "stroke": "#00649C",
              },
              ":disabled": Object {
                "backgroundColor": "#e5e5e5",
                "fill": "#e5e5e5",
              },
              "backgroundColor": "#ffffff",
              "border": "0.5px solid #848484",
              "borderRadius": "5px",
              "fill": "#848484",
              "stroke": "#848484",
            },
            "small": Object {
              "&:hover": Object {
                "backgroundColor": "#848484",
                "color": "#ffffff",
                "fill": "#ffffff",
              },
              "backgroundColor": "#fff",
              "border": "1px solid",
              "color": "#848484",
              "fill": "#848484",
            },
            "xsmall": Object {
              "&:hover": Object {
                "backgroundColor": "#00649C",
                "color": "#fff",
                "fill": "#fff",
              },
              "backgroundColor": "#fff",
              "border": "1px solid #00649C",
              "color": "#00649C",
              "fill": "#00649C",
            },
          },
          "inputSize": Object {
            "lg": Object {
              "fontSize": "28px",
              "lineHeight": "36px",
              "padding": "10px 10px",
            },
            "md": Object {
              "-moz-lineHeight": "20px",
              "fontSize": "16px",
              "lineHeight": "20px",
              "padding": "10px 10px",
            },
            "sm": Object {
              "fontSize": "16px",
              "lineHeight": "20px",
              "padding": "4px 10px",
            },
            "xs": Object {
              "fontSize": "16px",
              "lineHeight": "18px",
              "padding": "0 10px",
            },
          },
          "inputVariant": Object {
            "default": Object {
              "border": "1px solid #848484",
              "borderRadius": "5px",
              "color": "#333333",
              "rs": Object {
                "control": Object {
                  "border": "1px solid #848484",
                  "borderRadius": "5px",
                  "color": "#333333",
                },
                "dropdownIndicator": Object {},
                "indicatorSeparator": Object {},
                "menu": Object {},
                "option": Object {},
                "selectContainer": Object {},
                "singleValue": Object {},
                "valueContainer": Object {},
              },
            },
            "firstField": Object {
              "border": "1px solid #848484",
              "borderRadius": "5px 5px 0 0",
              "color": "#333333",
              "rs": Object {
                "control": Object {
                  "border": "1px solid #848484",
                  "borderRadius": "5px 5px 0 0",
                  "color": "#333333",
                },
                "dropdownIndicator": Object {},
                "indicatorSeparator": Object {},
                "menu": Object {},
                "option": Object {},
                "selectContainer": Object {},
                "singleValue": Object {},
                "valueContainer": Object {},
              },
            },
            "lastField": Object {
              "border": "1px solid #848484",
              "borderRadius": "0 0 5px 5px",
              "color": "#333333",
              "rs": Object {
                "control": Object {
                  "border": "1px solid #848484",
                  "borderRadius": "0 0 5px 5px",
                  "color": "#333333",
                },
                "dropdownIndicator": Object {},
                "indicatorSeparator": Object {},
                "menu": Object {},
                "option": Object {},
                "selectContainer": Object {},
                "singleValue": Object {},
                "valueContainer": Object {},
              },
            },
            "primary": Object {
              "border": "1px solid #848484",
              "borderRadius": "0",
              "color": "#333333",
              "rs": Object {
                "control": Object {
                  "border": "1px solid #848484",
                  "borderRadius": "0",
                  "color": "#333333",
                },
                "dropdownIndicator": Object {},
                "indicatorSeparator": Object {},
                "menu": Object {},
                "option": Object {},
                "selectContainer": Object {},
                "singleValue": Object {},
                "valueContainer": Object {},
              },
            },
            "secondary": Object {},
          },
          "text": Object {
            "document": Object {
              "color": "#4F4F4F",
              "fontFamily": "'Museo Sans 300'",
              "fontSize": "14px",
              "lineHeight": "20px",
              "marginBottom": "10px",
              "marginTop": "10px",
              "textAlign": "left",
            },
            "documentTitle": Object {
              "color": "#00649C",
              "fontFamily": "'Museo Sans 500'",
              "fontSize": "18px",
              "lineHeight": "24px",
              "textAlign": "center",
            },
            "firstHeader": Object {
              "color": "#333333",
              "fontFamily": "'Museo Sans 700'",
              "fontSize": "18px",
              "lineHeight": "30px",
              "marginBottom": "10px",
              "marginTop": "10px",
              "textAlign": "left",
            },
            "secondHeader": Object {
              "color": "#333333",
              "fontFamily": "'Museo Sans 500'",
              "fontSize": "14px",
              "lineHeight": "20px",
              "marginBottom": "10px",
              "marginTop": "10px",
              "textAlign": "left",
            },
          },
        },
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "data": Object {
          "message": "1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <ProfileNotification
          data={
            Object {
              "message": "1",
            }
          }
        />,
        "theme": Object {
          "borderColor": Object {
            "color0": "#FFFFFF",
            "color1": "#E5E5E5",
            "color10": "#AEAEAE",
            "color11": "#333333",
            "color12": "#DF4624",
            "color13": "#f2f2f2",
            "color14": "#fff3c1",
            "color2": "#DEDEDE",
            "color3": "#4F4F4F",
            "color4": "#848484",
            "color5": "#007FAF",
            "color6": "#006699",
            "color7": "#00649C",
            "color8": "#007CC4",
            "color9": "#F97E63",
          },
          "borderRadius": Array [
            0,
            2,
            4,
            8,
            12,
            16,
            18,
            20,
            24,
            28,
            32,
            36,
            40,
            44,
            48,
            52,
            56,
            60,
            64,
            68,
            72,
            76,
            80,
          ],
          "boxShadow": Array [
            "2px 4px 8px 0px rgba(138,138,138,0.5)",
            "2px 4px 8px 0px rgba(138,138,138,0.5)",
            "2px 2px 4px 0px rgba(0,127,175,1)",
          ],
          "breakpoints": Array [
            "1024px",
            "1200px",
            "1440px",
          ],
          "colors": Object {
            "color0": "#FFFFFF",
            "color1": "#E5E5E5",
            "color10": "#AEAEAE",
            "color11": "#333333",
            "color12": "#DF4624",
            "color13": "#f2f2f2",
            "color14": "#fff3c1",
            "color2": "#DEDEDE",
            "color3": "#4F4F4F",
            "color4": "#848484",
            "color5": "#007FAF",
            "color6": "#006699",
            "color7": "#00649C",
            "color8": "#007CC4",
            "color9": "#F97E63",
          },
          "fontFamily": Object {
            "primary300": "Museo Sans 300",
            "primary500": "Museo Sans 500",
            "primary700": "Museo Sans 700",
            "secondary": "Circe Regular",
            "secondaryBold": "Circe Bold",
          },
          "fontSizes": Array [
            0,
            2,
            4,
            8,
            12,
            16,
            18,
            20,
            24,
            28,
            32,
            36,
            40,
            44,
            48,
            52,
            56,
            60,
            64,
            68,
            72,
            76,
            80,
          ],
          "fontWeight": Array [
            300,
            500,
            700,
          ],
          "lineHeight": Array [
            0,
            2,
            4,
            8,
            12,
            16,
            18,
            20,
            24,
            28,
            32,
            36,
            40,
            44,
            48,
            52,
            56,
            60,
            64,
            68,
            72,
            76,
            80,
          ],
          "space": Array [
            0,
            2,
            4,
            8,
            12,
            16,
            18,
            20,
            24,
            28,
            32,
            36,
            40,
            44,
            48,
            52,
            56,
            60,
            64,
            68,
            72,
            76,
            80,
          ],
          "textAlign": Array [
            "left",
            "center",
            "right",
          ],
          "variant": Object {
            "buttonComment": Object {
              "comment": Object {
                "border": 0,
                "fill": "#006699",
                "minWidth": "20px",
              },
              "emptyComment": Object {
                "border": 0,
                "fill": "#848484",
                "minWidth": "20px",
              },
              "newComment": Object {
                "border": 0,
                "fill": "#df4624",
                "minWidth": "20px",
              },
            },
            "buttonSize": Object {
              "large": Object {
                "fontFamily": "Circe Bold",
                "fontSize": " 32px",
                "lineHeight": "40px",
                "minWidth": "500px",
                "padding": "8px 60px",
              },
              "medium": Object {
                "fontFamily": "Circe Bold",
                "fontSize": "18px",
                "lineHeight": "24px",
                "minWidth": "250px",
                "padding": "8px 0",
              },
              "small": Object {
                "fontFamily": "Circe Regular",
                "fontSize": " 16px",
                "lineHeight": "16px",
                "padding": "2px 0",
              },
              "toApproval": Object {
                "fontSize": "15px",
                "padding": "2px",
              },
              "xsmall": Object {
                "-moz-padding": "0",
                "fontSize": " 14px",
                "fontWeight": 700,
                "lineHeight": "16px",
                "minWidth": "70px",
                "padding": "2px 0",
              },
            },
            "buttons": Object {
              "disabled": Object {
                "&:hover": Object {
                  "backgroundColor": "#00649c57",
                  "color": "#fff",
                  "fill": "#fff",
                },
                "backgroundColor": "#fff",
                "border": "1px solid #00649c57",
                "color": "#00649c57",
                "fill": "#00649c57",
              },
              "empty": Object {
                "&:hover": Object {
                  "color": "#00649C",
                  "fill": "#00649C",
                  "stroke": "#00649C",
                },
                ":disabled": Object {
                  "backgroundColor": "#e5e5e5",
                  "fill": "#e5e5e5",
                },
                "backgroundColor": "#ffffff",
                "border": 0,
                "fill": "#848484",
                "stroke": "#848484",
              },
              "large": Object {
                "&:hover": Object {
                  "backgroundColor": "#00649C",
                  "color": "#ffffff",
                  "fill": "#ffffff",
                },
                "backgroundColor": "transparent",
                "border": "2px solid",
                "color": "#00649C",
                "fill": "#00649C",
              },
              "outlineGray": Object {
                "&:hover": Object {
                  "borderColor": "#00649C",
                  "color": "#00649C",
                  "fill": "#00649C",
                  "stroke": "#00649C",
                },
                ":disabled": Object {
                  "backgroundColor": "#e5e5e5",
                  "fill": "#e5e5e5",
                },
                "backgroundColor": "#ffffff",
                "border": "0.5px solid #848484",
                "borderRadius": "5px",
                "fill": "#848484",
                "stroke": "#848484",
              },
              "small": Object {
                "&:hover": Object {
                  "backgroundColor": "#848484",
                  "color": "#ffffff",
                  "fill": "#ffffff",
                },
                "backgroundColor": "#fff",
                "border": "1px solid",
                "color": "#848484",
                "fill": "#848484",
              },
              "xsmall": Object {
                "&:hover": Object {
                  "backgroundColor": "#00649C",
                  "color": "#fff",
                  "fill": "#fff",
                },
                "backgroundColor": "#fff",
                "border": "1px solid #00649C",
                "color": "#00649C",
                "fill": "#00649C",
              },
            },
            "inputSize": Object {
              "lg": Object {
                "fontSize": "28px",
                "lineHeight": "36px",
                "padding": "10px 10px",
              },
              "md": Object {
                "-moz-lineHeight": "20px",
                "fontSize": "16px",
                "lineHeight": "20px",
                "padding": "10px 10px",
              },
              "sm": Object {
                "fontSize": "16px",
                "lineHeight": "20px",
                "padding": "4px 10px",
              },
              "xs": Object {
                "fontSize": "16px",
                "lineHeight": "18px",
                "padding": "0 10px",
              },
            },
            "inputVariant": Object {
              "default": Object {
                "border": "1px solid #848484",
                "borderRadius": "5px",
                "color": "#333333",
                "rs": Object {
                  "control": Object {
                    "border": "1px solid #848484",
                    "borderRadius": "5px",
                    "color": "#333333",
                  },
                  "dropdownIndicator": Object {},
                  "indicatorSeparator": Object {},
                  "menu": Object {},
                  "option": Object {},
                  "selectContainer": Object {},
                  "singleValue": Object {},
                  "valueContainer": Object {},
                },
              },
              "firstField": Object {
                "border": "1px solid #848484",
                "borderRadius": "5px 5px 0 0",
                "color": "#333333",
                "rs": Object {
                  "control": Object {
                    "border": "1px solid #848484",
                    "borderRadius": "5px 5px 0 0",
                    "color": "#333333",
                  },
                  "dropdownIndicator": Object {},
                  "indicatorSeparator": Object {},
                  "menu": Object {},
                  "option": Object {},
                  "selectContainer": Object {},
                  "singleValue": Object {},
                  "valueContainer": Object {},
                },
              },
              "lastField": Object {
                "border": "1px solid #848484",
                "borderRadius": "0 0 5px 5px",
                "color": "#333333",
                "rs": Object {
                  "control": Object {
                    "border": "1px solid #848484",
                    "borderRadius": "0 0 5px 5px",
                    "color": "#333333",
                  },
                  "dropdownIndicator": Object {},
                  "indicatorSeparator": Object {},
                  "menu": Object {},
                  "option": Object {},
                  "selectContainer": Object {},
                  "singleValue": Object {},
                  "valueContainer": Object {},
                },
              },
              "primary": Object {
                "border": "1px solid #848484",
                "borderRadius": "0",
                "color": "#333333",
                "rs": Object {
                  "control": Object {
                    "border": "1px solid #848484",
                    "borderRadius": "0",
                    "color": "#333333",
                  },
                  "dropdownIndicator": Object {},
                  "indicatorSeparator": Object {},
                  "menu": Object {},
                  "option": Object {},
                  "selectContainer": Object {},
                  "singleValue": Object {},
                  "valueContainer": Object {},
                },
              },
              "secondary": Object {},
            },
            "text": Object {
              "document": Object {
                "color": "#4F4F4F",
                "fontFamily": "'Museo Sans 300'",
                "fontSize": "14px",
                "lineHeight": "20px",
                "marginBottom": "10px",
                "marginTop": "10px",
                "textAlign": "left",
              },
              "documentTitle": Object {
                "color": "#00649C",
                "fontFamily": "'Museo Sans 500'",
                "fontSize": "18px",
                "lineHeight": "24px",
                "textAlign": "center",
              },
              "firstHeader": Object {
                "color": "#333333",
                "fontFamily": "'Museo Sans 700'",
                "fontSize": "18px",
                "lineHeight": "30px",
                "marginBottom": "10px",
                "marginTop": "10px",
                "textAlign": "left",
              },
              "secondHeader": Object {
                "color": "#333333",
                "fontFamily": "'Museo Sans 500'",
                "fontSize": "14px",
                "lineHeight": "20px",
                "marginBottom": "10px",
                "marginTop": "10px",
                "textAlign": "left",
              },
            },
          },
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "data": Object {
            "message": "1",
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
